### DISCUSSION TITLE
[Question] Can I maintain the selected text after selection?

### DISCUSSION THREAD

**Discussion by ghost** [Blazored/Typeahead#244] [Q&A]
Hi guys! 

Im implementing for the first time your component, and I have just one problem I would like to keep writing on the input after the selection to avoid writing the same text again.

Is it possible to do this?

Thanks!!!

**Comment by chrissainty** [comment]
Unfortunately, this isn't possible. The input where you type is replaced with the selected templates once you make a selection.
